c/datasetjson.c

```diff
@@ -1756,7 +1759,27 @@ void respondWithDatasetMetadata(HttpResponse *response) {
 
   int fieldCount = defaultCSIFieldCount;
   char **csiFields = defaultCSIFields;
-
+  int asteriskPos = indexOf(dsn,dsnLen,'*',0);
+  int dblAsteriskPos = indexOfString(dsn,dsnLen,"**",0);
+  int periodPos = lastIndexOf(dsn, dsnLen, '.');
+  int addQualifiers = !strcmp(addQualifiersArg, "true");
+  
+  if(addQualifiers){
```

Comment:
Please fix the white space problems. Check that you have spaces instead of tabs and that the indent is set to 2 instead of 4.